Examples/VisionExamples/Examples/ARCustomizationViewController.swift

```diff
@@ -0,0 +1,146 @@
+            // Also you may want to stop showing AR lane
+            visionARViewController.isLaneVisible = false
+            // Set the desired fence visibility distance in meters.
+            visionARManager.set(fenceVisibilityDistance: 100)
+        },
+        // Let's set a few more different values to see the difference
+        { visionARManager, visionARViewController in
+            visionARManager.set(fenceVisibilityDistance: 150)
+        }, { visionARManager, visionARViewController in
+            visionARManager.set(fenceVisibilityDistance: 250)
+        },
+       // To customize fence appearence you need to use FenceVisualParams structure.
+       // Its constructor inits all the fields with the values which is used
+       // by VisionARViewController by default
+        { visionARManager, visionARViewController in
+            // Create an instance of `FenceVisualParams`
+            let fenceVisualParams = FenceVisualParams()
+            // Set the color of the fence
+            fenceVisualParams.color = UIColor.green
+            // Set the height of the fence in meters
+            fenceVisualParams.height = 3
+            // Set the vertical offset of the fence above the road surface in meters
+            fenceVisualParams.verticalOffset = 0
+            // Set the horizontal offset of the fence from the center of the road in meters
+            fenceVisualParams.horizontalOffset = 5
+            // Set the number of arrows in the fence
+            fenceVisualParams.sectionsCount = 4
+            // After `fenceVisualParams` is configured you need to set it to the view controller
+            visionARViewController.set(fenceVisualParams: fenceVisualParams)
+        },
+        // Let's customize fence with different visual params to see the difference
+        { visionARManager, visionARViewController in
+            let fenceVisualParams = FenceVisualParams()
+            fenceVisualParams.color = UIColor.red
+            fenceVisualParams.height = 2
+            fenceVisualParams.verticalOffset = 1.5
+            fenceVisualParams.horizontalOffset = 2
+            fenceVisualParams.sectionsCount = 5
+            visionARViewController.set(fenceVisualParams: fenceVisualParams)
+        }, { visionARManager, visionARViewController in
+            let fenceVisualParams = FenceVisualParams()
+            fenceVisualParams.color = UIColor.brown
+            fenceVisualParams.height = 1.5
+            fenceVisualParams.verticalOffset = 0.5
+            fenceVisualParams.horizontalOffset = 3
+            fenceVisualParams.sectionsCount = 6
+            // Let's show lane as well
+            visionARViewController.isLaneVisible = true
+            visionARViewController.set(fenceVisualParams: fenceVisualParams)
+        }, { visionARManager, visionARViewController in
+            let fenceVisualParams = FenceVisualParams()
+            fenceVisualParams.color = UIColor.yellow
+            fenceVisualParams.height = 4
```

Comment:
To discuss:
the height of the fence is 3, then 2, then 1.5, then 4, while we iterating though the styles.

Should we sort it and have them in ascending order to show the difference between styles more smoothly? Does that make sense?